docs/scalar_tensor.rst

```diff
@@ -3,6 +3,33 @@
 Scalar fields
 =============
 
+**Taichi fields** are used to store data.
+
+Field **elements** could be either scalar, vector or matrix (see :ref:`matrix`).
+We'll only talk about **scalar fields** in this chapter.
+
+Field **dimensions** could be either 0D, 1D, 2D, 3D, ...
```

Comment:
We use `either` two lines above. Maybe replace it with other word can make the readers happier.
```suggestion
Field **dimensions** could be arbitrary, including but not limited to 0D, 1D, 2D...
```